apps/studio/lib/api/edgeFunctions.ts

```diff
@@ -0,0 +1,7 @@
+export const isValidEdgeFunctionURL = (url: string) => {
+  const regexValidEdgeFunctionURL = new RegExp(
+    '^https://[a-z]*.supabase.(red|co)/functions/v[0-9]{1}/.*$'
```

Comment:
just noticed this is slightly different from what @staaldraad added in the previous PR if there's a difference functionally?

'^https://[a-z]*.supabase.(red|co)/functions/v[0-9]{1}/\S*$',

```suggestion
    '^https://[a-z]*.supabase.(red|co)/functions/v[0-9]{1}/.*$'
```

Reply:
i think they should be the same yeah - tests are passing as expected so no issue